### PR TITLE
added members section

### LINKED ISSUE #173: Create Design for Members Section
### Description
<write your description> Create a design for Members section . Kindly go through the website  and make this in accordance with the present theme. The members section should contain the cards/details of the members of particular community. You should also make a design for mobile version.

### Screenshots/testimonials
<place your screenshots or testimonials here if applicable>

### PR DESCRIPTION
**Pull request by hariom1625**
### fixes #173 

Description: Added members section.

PS: I have added dummy info since I didn't have all the required details, will update it soon. You can review the PR after adding the actual details i will make it ready for merging.
#### Checklist
- [x] I've followed the Contributing guidelines provided in the repository.
- [x] I've made the changes which were demanded in the linked issue.
- [x] I've tested my code on a Chromium based browser.
- [x] I've tested my code on Mozilla Firefox.
- [x] My code gave a clean console on debugging. (no warnings/errors)

### CODE REVIEW

**Comment by ankiiitraj** [comment]
I see a lot of code 😨

**Comment by hariom1625** [comment]
> I see a lot of code 😨

There were already different components made for different communities so enduring that further, I tried to keep the files and details for different communities separate so it got a bit lengthy 😅

**Comment by ankiiitraj** [comment]
No issues! Great work btw!

**Comment by EmperorYP7** [comment]
Whats the progress on this? Do we have the details of all members?

**Comment by hariom1625** [comment]
> Whats the progress on this? Do we have the details of all members?

Not yet, maybe till tomorrow, I will try to add the details of all the members.

**Comment by hariom1625** [comment]
The PR can be reviewed now.
I have updated the details of most of the members and the remaining details we can add on the go.
Also,I am not able to figure out how to render the Github logo only for those members who have an account. If anyone has any idea can share or we can put up an issue for the same.

File: client/src/components/Members/Community/Head.members.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+import {ReactComponent as Email} from "../img/mail.svg";
+import planet from "../img/planet_2.svg";
+
+import {HeadMembersLists as headLists} from './HeadMembersList.js';
+
+function HeadMembers() {
+
+      return (<div className="head-div">
+            <h1 className="community-header-head">
+                  Community Members
```
Comment (EmperorYP7) [inline client/src/components/Members/Community/Head.members.js]: I think "Members" will be more relevant here. "Community Members" doesnt look good in mobile view anyways. Afterall, the main title should be according to the navbar.

File: client/src/components/Members/Community/Head.members.css [hunk 2]
```diff
@@ -0,0 +1,67 @@
+.community-header-head {
+      text-align: center;
+      color: #EE2A7B;
+      font-weight: 400;
+      margin-top: 50px;
+      font-size: 3rem;
```
Comment (EmperorYP7) [inline client/src/components/Members/Community/Head.members.css]: With reference to #198 and #204, change the font-size to be `6vh`.

**Review by EmperorYP7 — CHANGES_REQUESTED** [review]
Very well done @hariom1625! :rocket: Just a few minor tweaks.

**Comment by devansh-srivastav-kashyap** [comment]
Over-all, the website look appealing. Good work.

Put Joint Secretary just below Secretary.
Faculty is to be replaced by Faculty in Charge- Student Welfare

**Comment by ankiiitraj** [comment]
> Over-all, the website look appealing. Good work.
> 
> Put Joint Secretary just below Secretary.
> Faculty is to be replaced by Faculty in Charge- Student Welfare

Or put Joint Sec beside the Sec?

**Comment by hariom1625** [comment]
@ankiiitraj and @devansh-srivastav-kashyap Which layout is being finalized?

**Comment by devansh-srivastav-kashyap** [comment]
Website me toh ek k neeche ek hai na?

**Comment by devansh-srivastav-kashyap** [comment]
Atleast in mobile, ek ke neeche ek hai.. toh in reference of mobile view, put j sec just below Secretary

**Comment by ankiiitraj** [comment]
How about adding a border like this? @hariom1625 
Screenshot from 2020-11-04 00-54-34

File: client/src/components/Members/Community/Event.members.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
+// import {ReactComponent as Github} from "../img/github-brands.svg";
```
Comment (anktrj) [inline client/src/components/Members/Community/Event.members.js]: cleanup required!

File: client/src/components/Members/Community/EventMembersList.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+export const EventMembersLists = [{
+            id: 0,
+            img: './images/jaiveer.jfif',
+            name: 'Jaiveer Singh',
+            position: 'Team Head',
+            mail: 'mailto:1999jaiveersingh@gmail.com',
+            // github: '',
```
Comment (anktrj) [inline client/src/components/Members/Community/EventMembersList.js]: unnecssary comments needs to be removed before merging

**Review by ankiiitraj — APPROVED** [review]
LGTM 🎉 
Very neat work!

**Comment by hariom1625** [comment]
> How about adding a border like this? @hariom1625

This looks great !! Will add the border in the next commit :-)

**Review by tusharjain0022 — APPROVED** [review]
Beautiful 🤩 , Nice Work @hariom1625 🙌

**Review by EmperorYP7 — APPROVED** [review]
This is super clean code. Nice work @hariom1625! 🥳

**Comment by EmperorYP7** [comment]
Merging this against all reviewers' approval.

**Pull request merged**